cmake/setup_write_config.cmake

```diff
@@ -20,7 +20,8 @@
 #                                                                      #
 ########################################################################
 
-DEAL_II_QUERY_GIT_INFORMATION()
+SET(DEAL_II_GIT_PREFIX "DEAL_II")
+DEAL_II_QUERY_GIT_INFORMATION(DEAL_II_GIT_PREFIX)
```

Comment:
What about simply
```
DEAL_II_QUERY_GIT_INFORMATION("DEAL_II")
```

By introducing `DEAL_II_GIT_PREFIX` you otherwise indicate that this variable is used elsewhere (or that it is configurable).